### PR TITLE
Swift Package Manager support

### PR DESCRIPTION
**Pull request by michaelshass**
- Sets swift language to 5.1
- Update pod specs
- Change folder structure to SPM default

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// swift-tools-version:5.1
+
```
Comment (markuswntr) [inline Package.swift]: No empty line between tools version and package description

File: Package.swift [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import PackageDescription
+
+let package = Package(
+    name: "ModelDataSource",
+    platforms: [
+        .iOS(.v12), .tvOS(.v12)
```
Comment (markuswntr) [inline Package.swift]: watchOS? (not available?)

Comment (michaelshass) [reply Package.swift]: nope

**Comment by michaelshass** [comment]
> Deployment Target on iOS & tvOS is set to 11.0 - Maybe increase even further and bump to 12.0?

**Pull request merged**